tests/pmrfc3164-tagEndingByColon.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+# add 2016-11-22 by Pascal Withopf, released under ASL 2.0
+. $srcdir/diag.sh init
+. $srcdir/diag.sh generate-conf
+. $srcdir/diag.sh add-conf '
+module(load="../plugins/imtcp/.libs/imtcp")
+input(type="imtcp" port="13514" ruleset="customparser")
+parser(name="custom.rfc3164" type="pmrfc3164" force.tagEndingByColon="on")
+template(name="outfmt" type="string" string="-%syslogtag%-\n")
```

Comment:
I think we should change this test to also make sure that the MSG property is properly populated, e.g. by including %msg% in the template and in the success check.